server/src/main/java/io/crate/execution/dml/TranslogIndexer.java

```diff
@@ -138,7 +147,27 @@ private IndexDocumentBuilder populateLuceneFields(BytesReference source) throws
             entry.setValue(castValue);
         }
 
-        addIndexColumns(indexColumns, docBuilder);
+        for (var indexColumn : indexColumns) {
+            String fqn = indexColumn.reference().storageIdent();
+            for (var input : indexColumn.inputs()) {
+                Object value = input.value(docMap);
+                if (value == null) {
+                    continue;
+                }
```

Comment:
Can we add some tests that cover null values via `assertTranslogParses`?

Reply:
Thanks, pushed a null for array, top-column and sub-column, have a failure on sub-column that fails on assertTranslogParses but could be a test setup issue, checking

Reply:
Yep, it's unrelated to FT, simple sub-column with NULL fails on `assertTranslogParses` on master